scifin/timeseries/randomseries.py

```diff
@@ -389,13 +394,14 @@ def moving_average(start_date, end_date, frequency, cst, order, coeffs, sigma, t
                 x[t] -= coeffs[q] * a[t-q-1]
     
     # Compute theoretical values
-    V = 1.
-    for q in range(Q):
-        V += coeffs[q]**2
-    V *= sigma**2
-    print("The expected value for this MA(" + str(Q) + ") model is: " + str(cst))
-    print("The estimation of the variance for this MA(" + str(Q) + ") model is: " + str(V) + \
-          " , i.e. a standard deviation of: " + str(np.sqrt(V)) + "\n")
+    if verbose:
+        V = 1.
+        for q in range(Q):
+            V += coeffs[q]**2
+        V *= sigma**2
+        print("The expected value for this MA(" + str(Q) + ") model is: " + str(cst))
+        print("The estimation of the variance for this MA(" + str(Q) + ") model is: " + str(V) + \
+              " , i.e. a standard deviation of: " + str(np.sqrt(V)) + "\n")
```

Comment:
use f-string

Reply:
Done (for all functions in this module).